pages/apim/3.x/installation-guide/configuration/repositories/installation-guide-repositories-mongodb.adoc

```diff
@@ -53,31 +53,63 @@ management:
     dbname:                     # mongodb name (default gravitee)
     host:                       # mongodb host (default localhost)
     port:                       # mongodb port (default 27017)
+
+## Client settings
+    description:                # mongodb description (default gravitee.io)
     username:                   # mongodb username (default null)
     password:                   # mongodb password (default null)
-    authSource:                 # mongodb authentication source (default gravitee)
-    connectionPerHost:          # mongodb connection per host (default 10)
-    connectTimeOut:             # mongodb connection time out (default 0 -> never)
+    authSource:                 # mongodb authentication source (when at least a user or a password is defined, default gravitee)
+    readPreference:              # possible values are 'nearest', 'primary', 'primaryPreferred', 'secondary', 'secondaryPreferred'
+    readPreferenceTags:          # list of read preference tags (https://docs.mongodb.com/manual/core/read-preference-tags/#std-label-replica-set-read-preference-tag-sets)
+### Write concern
+    writeConcern:               # possible values are 1,2,3... (the number of node) or 'majority' (default is 1)
+    wtimeout:                   # (default is 0)
+    journal:                    # (default is true)
+
+## Socket settings
+    connectTimeOut:             # mongodb connection time out (default 1000)
+    socketTimeout:              # mongodb socket time out (default 1000)
+
+## Cluster settings
+    serverSelectionTimeout:     # mongodb server selection time out (default 1000)
```

Comment:
Could we please change "time out" to "timeout" in the definition for all of these? :) 

## Socket settings
    connectTimeOut:             # mongodb connection timeout (default 1000)
    socketTimeout:              # mongodb socket timeout (default 1000)